README.md

```diff
@@ -23,21 +23,105 @@
+  UX to efficient algorithms. The default storage backend uses Git repositories
+  for "physical storage", for wide interoperability and ease of onboarding.
+
+- **Mercurial & Sapling**: There are many Mercurial-inspired features, such as
+  the [revset] language to select commits. There is [no explicit index][no-index]
+  or "staging area". Branches are "anonymous" like Mercurial, so you don't need
+  to make up a name for each small change. Primitives for rewriting history are
+  powerful and simple. Formatting output is done with a robust template language
+  that can be configured by the user.
+
+- **Pijul & Darcs**: Jujutsu keeps track of conflicts as [first-class objects][conflicts]
+  in its model; they are first-class in the same way commits are, while
+  alternatives like Git simply think of conflicts as textual diffs. While not
+  as rigorous as patch-based systems like Pijul and Darcs (which formalize
+  the underlying theory of three-way merges), the effect is that many forms of
+  conflict resolution can be performed and propagated automatically.
+
+[perf]: https://github.com/martinvonz/jj/discussions/49
+[revset]: https://martinvonz.github.io/jj/latest/revsets/
+[no-index]: https://martinvonz.github.io/jj/latest/git-comparison/#the-index
+[conflicts]: https://martinvonz.github.io/jj/latest/conflicts/
+
+And it adds several innovative, useful features of its own:
+
+- **Working-copy-as-a-commit**: Changes to files are [recorded automatically][wcc]
+  as normal commits, and amended on every subsequent change. This "snapshot"
+  design simplifies the user-facing data model (commits are the only visible
+  object), simplifies internal algorithms, and completely subsumes features like
+  Git's stashes or the index/staging-area.
+
+- **Operation log & undo**: Jujutsu records every operation that is performed on the
+  repository, from commits, to pulls, to pushes. This makes debugging problems like
+  "what just happened?" or "how did I end up here?" easier, *especially* when
+  you're helping your coworker answer those questions about their repository!
+  And because everything is recorded, you can undo that mistake you just made
+  with ease. Version control has finally entered [the 1960s][undo-history]!
+
+- **Automatic rebase and conflict resolution**: When you modify a commit, every
+  descendent is automatically rebased on top of the freshly-modified one. This
+  makes "patch-based" workflows a breeze. If you resolve a conflict in a commit,
+  the _resolution_ of that conflict is also propagated through descendants as
+  well. This is effectively a more powerful version of `git rebase --update-
+  refs` combined with `git rerere` while being simpler and transparent to the
+  user.
+
+  > [!WARNING]
+  > The following features are experimental, and may have bugs or backwards
+  > incompatible storage and user-interface changes!
+
+- **Workspaces**: A workspace lets you make a copy of the current repository
```

Comment:
Are workspaces really that experimental?

I'm not sure we even need to mention them at all, but if we do, we could say that they are analogous to Git worktrees.

Reply:
Once it exists, `jj run` would be a great item to put here.

Reply:
Ah, I didn't realize it was really that close to `worktree`. Though, I have my own idea about what workspaces might entail, I should write it up sometime...

Reply:
> we could say that they are analogous to Git worktrees.

That doesn't help those of us that don't use Git.

Reply:
> That doesn't help those of us that don't use Git.

If we do mention them, I think we should both explain what they are *and* mention that there is an analogous Git feature.